rest/bulk_api.go

```diff
@@ -244,6 +244,37 @@ func (h *handler) handleDump() error {
 	return nil
 }
 
+
+// HTTP handler for _repair
+func (h *handler) handleRepair() error {
+
+	base.LogTo("HTTP", "Repair bucket")
+
+	// Todo: is this actually needed or does something else in the handler do it?  I can't find that..
+	defer h.requestBody.Close()
+
+	body, err := h.readBody()
+	if err != nil {
+		return err
+	}
+
+	repairBucketParams := db.RepairBucketParams{}
+	if err := json.Unmarshal(body, &repairBucketParams); err != nil {
+		return fmt.Errorf("Error unmarsalling %v into RepairJobParams.  Err: %v", string(body), err)
+	}
+
+	repairBucket := db.NewRepairBucket(h.db.Bucket)
+
+	repairBucket.InitFrom(repairBucketParams)
+
+	_, repairDocsErr := repairBucket.RepairBucket()
```

Comment:
I feel like the REST call should return details about what was changed.  At minimum a count, better would be a list of modified docs, even better would be a list of modified docs and a short summary the repair operation performed.

Reply:
Fixed, with an asterisk.  See the "Hack:" explanation.